bellboy/actors/lead.py

```diff
@@ -35,6 +39,14 @@ def startBellboyLead(self):
         self.ultrasonic_sensor = self.createActor(
             UltrasonicActor, globalName="ultrasonic"
         )
+        self.comms_actor = self.createActor(CommsActor, globalName="comms")
+        self.realtime_actor = self.createActor(
+            RealtimeCommsActor, globalName="realtime"
+        )
+
+        # requests to setup actors
+        self.send(self.comms_actor, CommsReq.AUTHENTICATE)
+        self.send(self.realtime_actor, Request.START)
         self.lcd = self.createActor(LcdActor, globalName="lcd")
```

Comment:
maybe move the lcd's creation to be with the others

Reply:
Moved.